form.php

```diff
@@ -1,3 +1,50 @@
+<?php
+require_once 'functions.php';
+
+$database = getDB();
```

Comment:
Move this to only happen if you need a db, currently happening on every page load. Probably best to move inside the issets of post